.github/workflows/test-sigstore.yml

```diff
@@ -0,0 +1,38 @@
+name: Run Sigstore Signer tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
should remove this as it will unfortunately keep failing for PRs

We should also maybe file an issue for managing these tests that can be only run on upstream branches... so that an issue would be raised when a test fails

Reply:
oh, I actually did that in kms-tests: you can probably copy the file-ticket-on-failure step from there

Reply:
> should remove this as it will unfortunately keep failing for PRs

Is that the case? Shouldn't it work, once the workflow is merged?

Reply:
I don't think so: pull requests have read only permissions and you need `id-token: write`

or in more OIDC terms, pull requests aren't allowed to "authenicate" as the project (even though sometimes it would make sense, since the authentication is not for the whole project but for the specific workflow which could be dedicated for PRs...)

Reply:
but feel free to test by merging: we can fix with another PR if it does not work

Reply:
@jku: Looks like this actually works:
https://github.com/secure-systems-lab/securesystemslib/actions/runs/4382839569

Build was triggered via pull request and apparently had the required permissions.

Or maybe it only worked because the pr didn't come from a fork. 🤷

Reply:
> maybe it only worked because the pr didn't come from a fork. shrug

yes, correct: I expect branches in upstream repo to work, but branches in forks  to fail